crates/api/build.rs

```diff
@@ -0,0 +1,7 @@
+fn main() -> std::io::Result<()> {
+    std::env::set_var("PROTOC", protobuf_src::protoc());
+    prost_build::Config::new()
+        .bytes(["."])
+        .compile_protos(&["src/wire.proto"], &["src/"])?;
```

Comment:
I feel that a specific directory for protobuf definitions makes sense, what do you think?

Reply:
Done: https://github.com/holochain/kitsune2/pull/38/commits/c7c3632db3f838143cf3b0a1ea4583f0ffb04bb2